### PR TITLE
Highlighter Task Lint Fixes

### PR DESCRIPTION
**Pull request by simensta**
Fixes most of the lint errors for the Highlighter task files. For ease of PR review, I wanted to merge these changes before addressing the Edge/IE issues mentioned in PR #3738. 

# Review Checklist

- [ ] Does it work in all major browsers: Firefox, Chrome, Edge, Safari?
- [ ] Does it work on mobile?
- [x] Can you `rm -rf node_modules/ && npm install` and app works as expected?
- [x] Did you deploy a staging branch? https://highlighter-lint-fixes.pfe-preview.zooniverse.org/
- [ ] Did you get any type checking errors from [Babel Typecheck](https://github.com/codemix/babel-plugin-typecheck)

## Optional

- [ ] If it's a new component, is it in ES6? Is it clear of warnings from ESLint?
- [ ] Have you replaced any `ChangeListener` or `PromiseRenderer` components with code that updates component state?
- [ ] If changes are made to the classifier, does the dev classifier still work?
- [ ] Have you added in [flow type annotations](https://flowtype.org/docs/type-annotations.html)?

### CODE REVIEW

File: app/classifier/tasks/highlighter/index.jsx [hunk 1]
```diff
@@ -1,3 +1,5 @@
+/* eslint no-underscore-dangle: ["error", { "allow": ["_toolIndex"] }] */
```
Comment (simensta) [inline app/classifier/tasks/highlighter/index.jsx]: This eslint rule modification could be added to the `.eslintrc` file.

File: app/classifier/tasks/highlighter/index.jsx [hunk 2]
```diff
@@ -143,6 +132,9 @@ Highlighter.defaultProps = {
 };
 
 Highlighter.propTypes = {
```
Comment (eatyourgreens) [inline app/classifier/tasks/highlighter/index.jsx]: Is it worth listing the properties in alphabetical order here?

Comment (simensta) [inline app/classifier/tasks/highlighter/index.jsx]: Definitely, that's an easy fix.

Comment (simoneduca) [inline app/classifier/tasks/highlighter/index.jsx]: Is this ready to go or are we waiting on changing that order?

Comment (simensta) [inline app/classifier/tasks/highlighter/index.jsx]: Just made the changes in 5f56936.

**Comment by simoneduca** [comment]
I've looked for missed parens, typos, etc. and seems good. Still I haven't followed this PR so I'm not sure what to test. Does the new highlighter tool still work after these fixes? Did anything else break?

**Pull request merged**